docs/rules.rst

```diff
@@ -143,3 +143,78 @@ Getting Ready to Train
 ======================
 
 Once you've written a few scoring rules, it's time for :doc:`training`. This computes optimal weighting coefficients and biases, which let the ruleset compute accurate confidences of its decisions. Be sure to write at least 2 rules before attempting to train; when there's only one, the trainer has nothing to balance.
+
```

Comment:
I think, if we're going to talk about trainees, it should be up under "Starting Your Ruleset", where I kind of half-bakedly already do it. Maybe, with enough conceptual explanation of the trainee data structure, a link to the code will suffice rather than another inevitably drifting excerpt?

Reply:
I agree about pointing toward the example code. I think adding an additional highlight of the `trainees` object in "Starting Your Ruleset" should be enough. We're telling them to use the example as a starting point, so they _should_ wind up reading that code (which explains the `trainees` object) anyway.

Random thought: Perhaps a `fathom-init` tool could be useful for setting up a directory structure and the skeleton of the trainees file.

Reply:
I have kicked around a "fathom-init" as well. :-) I'd take one. A few things get made implicitly, like `vectors` and `runs`, but it wouldn't be a bad thing to have `samples` , `samples/{validation,testing,training,unused}`, `rubrics.md`, and the example `rulesets.js` laid down. And we could automate `.gitattributes`. Hmm, I'm liking this more and more. FIts my usual evolutionary narrative:

1. Do it by hand.
2. Document it.
3. Script it.
4. Make it unnecessary altogether. :-)